app/assets/javascripts/client/inventories/InviteInventoryUserCtrl.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').controller('InviteInventoryUserCtrl', InviteInventoryUserCtrl);
+
+  InviteInventoryUserCtrl.$inject = ['$scope', 'InventoryInvitation'];
+  function InviteInventoryUserCtrl($scope, InventoryInvitation) {
+    var vm = this;
+    vm.sendInvitation = function(invitation) {
+      vm.alerts  = [];
+      vm.addAlert = function(message) {
```

Comment:
We should pull these variables and functions out of the nested `sendInvitation` function.